sass/abstracts/_variables.scss

```diff
@@ -0,0 +1,32 @@
+//COLORS
+
+$color-primary: #0e9900; //ok
```

Comment:
what are these `//ok` that you have placed in some lines?

Reply:
if _home.scss is not used then there is not need to be present in this pull request. 
Tip: only add files that is needed or is used by the project. Even if you think this file will be used add it only anly proper type arrives.

Reply:
I fixed all the errors. Thanks for the feedback